Source/Core/Common/QoSSession.h

```diff
@@ -0,0 +1,72 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <enet/enet.h>
+
+namespace Common
+{
+class QoSSession
+{
+public:
+  // 1 0 1 1 1 0 0 0
+  // DSCP      ECN
+  constexpr static int ef_tos = 0b10111000;
+
+  QoSSession() = default;
+  ~QoSSession();
+
+  QoSSession(ENetPeer* peer, int tos_val = ef_tos);
```

Comment:
This should be placed above the destructor. Also does it make sense to have `tos_val` as a constructor argument when this change doesn't pass anything different to it at any of the usage cases? Is there a scenario where it would be reasonable to change the value here?

Reply:
Since the `QoSSession` is now in Common I guess there could be more use cases for it than just setting the priority to EF.